lib/core/src/display/scene/htmlobject.rs

```diff
@@ -0,0 +1,49 @@
+use super::Object;
+use crate::system::web::{create_element_as, Result};
+use nalgebra::Vector3;
+use web_sys::HtmlElement;
+
+#[derive(Shrinkwrap)]
+#[shrinkwrap(mutable)]
+pub struct HTMLObject {
+    #[shrinkwrap(main_field)]
+    pub object: Object,
+    pub element: HtmlElement,
+    pub dimension: Vector3<f32>,
+}
+
+impl HTMLObject {
+    pub fn new(name: &str) -> Result<Self> {
+        let element = create_element_as::<HtmlElement>(name)?;
+        Ok(Self::from_element(element))
+    }
+
+    pub fn from_element(element: HtmlElement) -> Self {
+        let style = element.style();
+        style.set_property("transform-style", "preserve-3d").expect("transform-style: preserve-3d");
+        style.set_property("position", "absolute").expect("position: absolute");
+        style.set_property("width", "0px").expect("width: 0px");
+        style.set_property("height", "0px").expect("height: 0px");
```

Comment:
What these `.expect(...)` do? I mean, they throw custom panic, right? If so, what is the exact message of panic thrown here for example?